lsmb-request.pl

```diff
@@ -19,7 +19,7 @@ =head1 COPYRIGHT
 =cut
 
 package LedgerSMB::Handler;
-
+use lib '.';
```

Comment:
you would want to use FindBin, but it is also currently broken on Debian so......